tests/infra_test.go

```diff
@@ -53,17 +53,23 @@ import (
 )
 
 var _ = Describe("Infrastructure", func() {
-	tests.FlagParse()
+	var virtClient kubecli.KubevirtClient
+	var aggregatorClient *aggregatorclient.Clientset
+	var err error
 
-	virtClient, err := kubecli.GetKubevirtClient()
-	tests.PanicOnError(err)
+	BeforeEach(func() {
+		virtClient, err = kubecli.GetKubevirtClient()
+		tests.PanicOnError(err)
 
-	config, err := kubecli.GetConfig()
-	if err != nil {
-		panic(err)
-	}
+		if aggregatorClient == nil {
```

Comment:
Why do you only once fetch an aggregatorClient, but eveery time a KubevirtClient?

Reply:
`KubevirtClient` is protected and is fetched only once (it is a singletone).
This one is not.
I tried to follow the original flow as close as possible to avoid surprises (e.g. that this can change between tests).

Reply:
Great, thanks for the explanation!